### PR TITLE
set a new route structure to handle access components

### PR DESCRIPTION
**Pull request by aeddaqqa**
- moving the login page to the Camino suite application
- add new layout for access view
- add a new route for each access component
- add navigation function from suite application to wallet components

### CODE REVIEW

File: host/src/views/access/LoadComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import React, { useRef, useEffect } from "react";
+
+const LoadComponent = ({ type, props }) => {
+  const ref = useRef(null);
+  useEffect(() => {
+    mountAccessComponents(ref.current, type, props);
+  }, []);
+
+  return (
+    <div>
```
Comment (Chriserus) [inline host/src/views/access/LoadComponent.tsx:11]: Isn't it better to use React fragment for such cases?
```
<>
  <div ref={ref] />
</>
```

Comment (Chriserus) [reply host/src/views/access/LoadComponent.tsx:11]: It would clean up DOM a little bit

File: host/src/views/access/MountAccessComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useAppDispatch, useAppSelector } from "../../hooks/reduxHooks";
+import { changeActiveApp, getActiveApp } from "../../redux/slices/app-config";
+import LoadComponent from "./LoadComponent";
+const MountAccessComponent = ({ type }) => {
+  const navigate = useNavigate();
+  const [logged, setLogged] = useState(null);
```
Comment (Chriserus) [inline host/src/views/access/MountAccessComponent.tsx:8]: Shouldn't `logged` be a boolean, so, `false` as initial state?

File: wallet/src/App.vue [hunk 3]
```diff
@@ -54,12 +54,12 @@ export default {
...
-        if (locale) {
-            this.$root.$i18n.locale = locale
-        }
-        if (router.currentRoute.path === '/') router.push('/wallet')
-
+        // let locale = localStorage.getItem('lang')
+        // if (locale) {
+        //     this.$root.$i18n.locale = locale
+        // }
+        // if (router.currentRoute.path === '/') router.push('/wallet')
```
Comment (Chriserus) [inline wallet/src/App.vue]: Please delete commented out code. Git will remember!

File: wallet/src/moutHomePage.ts [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import Vue from 'vue'
```
Comment (Chriserus) [inline wallet/src/moutHomePage.ts]: Typo in file name `moutHomePage` ->  `mountHomePage`

File: wallet/src/router/index.ts [hunk 5]
```diff
@@ -41,41 +41,41 @@ const ifAuthenticated = (to: Route, from: Route, next: Function) => {
...
+    //         },
+    //         {
+    //             path: 'account/:index',
+    //             component: Account,
+    //             name: 'Account',
+    //         },
+    //     ],
+    //     component: Access,
+    //     beforeEnter: ifNotAuthenticated,
+    // },
```
Comment (Chriserus) [inline wallet/src/router/index.ts]: Delete commented out code if possible

File: wallet/src/views/access/mountAccessComponents.ts [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+            const context = {
+                props: props,
+            }
+            return createElement(mountComponent(type), context)
+        },
+    })
+    app.$mount(el)
+}
+
+// mount("#app");
```
Comment (Chriserus) [inline wallet/src/views/access/mountAccessComponents.ts]: Delete if possible

**Review by Chriserus — CHANGES_REQUESTED** [review]

File: host/src/views/access/LoadComponent.tsx (hunk 1 above)
Comment (aeddaqqa) [reply host/src/views/access/LoadComponent.tsx:11]: we need reference in the dom where the component should mount

File: host/src/views/access/MountAccessComponent.tsx (hunk 2 above)
Comment (aeddaqqa) [reply host/src/views/access/MountAccessComponent.tsx:8]: logged contained all the information of the wallet

**Review by Chriserus — APPROVED** [review]
LGTM

**Pull request merged**